internal/ingress/annotations/authtls/main.go

```diff
@@ -80,6 +81,12 @@ var authTLSAnnotations = parser.Annotation{
 			Risk:          parser.AnnotationRiskLow,
 			Documentation: `This annotation defines if the received certificates should be passed or not to the upstream server in the header "ssl-client-cert"`,
 		},
+		annotationAuthTLSPassCertToUpstreamHeader: {
+			Validator:     parser.ValidateNull,
```

Comment:
Please don't ignore the validation, as you are passing a string, you should add what is the possible value

Reply:
Yeah, I thought so too. Is there any existing example I can follow?